WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -172,6 +172,18 @@ private extension StoreOnboardingViewModel {
                         } else {
                             return true
                         }
+                    }).map({ [weak self] task in
+                        // If store is already live and launchStore task is incomplete
+                        // mark the task as complete
+                        guard let self,
+                              case .launchStore = task.type,
+                              !task.isComplete,
+                              let isBlogPublic = self.stores.sessionManager.defaultSite?.isBlogPublic,
+                              isBlogPublic else {
```

Comment:
nit: can it be simplified as
```suggestion
                              self.stores.sessionManager.defaultSite?.isBlogPublic == true else {
```

Reply:
Done in 674afb651457dee96d48c97a99b698474155f007